src/main/java/com/dataloom/data/storage/CassandraEntityDatastore.java

```diff
@@ -137,6 +137,20 @@ public EntitySetData getEntitySetData(
 
 
     @Override
+    public SetMultimap<FullQualifiedName, Object> getEntity(
+            UUID entitySetId,
+            UUID syncId,
+            String entityId,
+            Map<UUID, PropertyType> authorizedPropertyTypes ) {
+        ResultSet entity = session.execute( entitySetQuery.bind()
+                .setUUID( CommonColumns.ENTITY_SET_ID.cql(), entitySetId )
+                .setString( CommonColumns.ENTITYID.cql(), entityId )
+                .setSet( CommonColumns.PROPERTY_TYPE_ID.cql(), authorizedPropertyTypes.keySet() )
+                .setUUID( CommonColumns.SYNCID.cql(), syncId ) );
+        return RowAdapters.entity( entity, authorizedPropertyTypes, mapper );
```

Comment:
there's an `asyncLoadEntity` method in there, so you can do `asyncLoadEntity( ... ).getUninterruptibly()` to avoid code duplication